src/test/java/uk/gov/hmcts/juror/api/moj/service/JurorManagementServiceImplTest.java

```diff
@@ -180,7 +242,7 @@ public void test_reassignJuror_bureauUser_validRequest_happy() {
             .getPoolMemberSequenceNumber(anyString());
         verify(jurorHistoryService, times(1))
             .createReassignPoolMemberHistory(any(), any(), any());
-        verify(printDataService, times(1)).printConfirmationLetter(any());
```

Comment:
is this correct, as this a bureau user and has a police check would we expect this to be triggered?

Reply:
This is the expectation as the juror is in the responded state and has a police check so needs a letter to confirm the new pool allocated - I am also checking with Liam on this

Reply:
This is correct as confirmed with Liam